avatars/avatars.js

```diff
@@ -81,6 +84,11 @@ const textEncoder = new TextEncoder();
 const maxIdleVelocity = 0.01;
 const maxEyeTargetTime = 2000;
 
+const simplex = new SimplexNoise();      
```

Comment:
We can move this winds code to a separate file to make the code cleaner. Avatars.js update function is already far too long.

Reply:
[move wind to simulation](https://github.com/webaverse/app/pull/3029/commits/4d7328e91cffc16fb3b727f991f0d77fb1237ee9)